portfolio/src/main/webapp/index.html

```diff
@@ -18,8 +18,9 @@
             <a href="gallery.html"><img class="button" src="images/Button_Gallery.png"/></a>
         </div>
 
+        <br>
         <center>
-            <a href="form.html"><button class="buttonBox"><h3>Form Page</h3></button></a>
+            <a href="form.html"><button class="buttonBox button01"><h3>Comments!</h3></button></a>
```

Comment:
Not related with this PR (you can just update it in a follow up PR), but try give meaningful names, e.g "commentButton" or other at your choice.